index.html

```diff
@@ -136,85 +149,99 @@ <h4>2015</h4>
-                <li class="tech_lang">
-                    <h4>Skills</h4>
-                    <img src="images/Icon-Chevron-Right.png" alt="" srcset="">
-                </li>
-            </ul>
-        </section>
+            </section>
+        </div>
     </section>
     <section class="contact">
         <div class="contact-wrap">
             <img src="images/contact_shapes.png" alt="" srcset="" class="contact-shapes-bg">
-            <h4 class="contact-title">Contact me</h4>
-            <p class="contact-desc">
-                If you have an application you are interested in developing, a feature that you need built or a project
-                that needs coding.
-                I’d love to help with it!
-            </p>
-            <form action="https://formspree.io/f/xwkjrjye" method="post" class="contact-form">
-                <ul>
-                    <li>
-                        <input type="text" id="name" name="user_name" placeholder="Lorem Ipsum" maxlength="30" required/>
-                    </li>
-                    <li>
-                        <input type="email" id="mail" name="user_email" placeholder="loremipsum@gmail.com"  required />
-                    </li>
-                    <li>
-                        <textarea id="msg" name="user_message" maxlength="500" required>Write your message here...</textarea>
-                    </li>
-                    <li>
-                        <input type="submit" value="Get in touch">
-                    </li>
-                </ul>
-            </form>
-            <div class="contact-bt-line"></div>
+            <div class="contact-container">
+                <h4 class="contact-title">Contact me</h4>
+                <p class="contact-desc">
+                    If you have an application you are interested in developing, a feature that you need built or a
+                    project
+                    that needs coding.
+                    I’d love to help with it!
+                </p>
+                <form action="https://formspree.io/f/xwkjrjye" method="post" class="contact-form">
+                    <ul>
+                        <li>
+                            <input type="text" id="name" name="user_name" placeholder="Lorem Ipsum" maxlength="30"
+                                required />
+                        </li>
+                        <li>
+                            <input type="email" id="mail" name="user_email" placeholder="loremipsum@gmail.com"
+                                required />
+                        </li>
+                        <li>
+                            <textarea id="msg" name="user_message" maxlength="500"
+                                required>Write your message here...</textarea>
+                        </li>
+                        <li>
+                            <input type="submit" value="Get in touch">
+                        </li>
+                    </ul>
+                </form>
```

Comment:
- [ ] Your page looks amazingly responsive :clap: except that I think there's a slight change that can make the user interface more compelling so much that it enhances user experience across all devices. If you check the mobile design and the desktop design, you will notice the `contact form` is aligned at the center just as the `contact title` and `contact description`. Kindly align the form centrally just as you did in `width > 768px`, and see how interesting it gets :smile: 
- Here's the look at `wdith < 768px`:

![image](https://user-images.githubusercontent.com/99715640/230648653-bd7a729e-57ed-4784-9720-6b1c3b7b479f.png)